Source/WTF/wtf/text/Base64.h

```diff
@@ -86,11 +82,6 @@ Vector<uint8_t> base64URLEncodeToVector(std::span<const uint8_t>);
 String base64URLEncodeToString(std::span<const std::byte>);
 String base64URLEncodeToString(std::span<const uint8_t>);
 
-std::optional<Vector<uint8_t>> base64URLDecode(StringView);
-inline std::optional<Vector<uint8_t>> base64URLDecode(ASCIILiteral literal) { return base64URLDecode(StringView { literal }); }
-std::optional<Vector<uint8_t>> base64URLDecode(std::span<const std::byte>);
-std::optional<Vector<uint8_t>> base64URLDecode(std::span<const uint8_t>);
```

Comment:
What’s the rationale for removing these?

Reply:
it felt a bit unnecessary to have these where the only difference was whether `Base64DecodeMode::URL` was provided, but i can put them back if we prefer having the convenience of these utility methods